### DISCUSSION TITLE
ScalaJS questions: Dependencies and the 'sjs' suffix.

### DISCUSSION THREAD

**Discussion by roman-mibex-2** [com-lihaoyi/mill#3151] [Q&A] [answered]
Hi

I'm experimenting with Mill on a complex build (porting from SBT), and I don't get the ScalaJS dependencies correct:

In SBT, it resolves ScalaJS dependencies with the `_sjs1_3` suffix for ScalaJS in Maven repos. I don't know how to 'teach' Mill to add that suffix. It seems to not add the '_sjs1_3' suffix for ScalaJS dependencies.  My work around is to currently specify it explicitly:
   ```
   ivy"io.circe:circe-scalajs_sjs1_3:0.14.6"
   ```
   Am I missing something obvious to add that 'sjs' for ScalaJS builds? 
   I certainly can create a helper function/method, and do it myself. But I feel like I'm maybe missing the obvious.

**Comment by lefou** [comment, marked as answer]
Please read the following page: https://mill-build.com/mill/Library_Dependencies.html

Especially the section about [ScalaJS dependencies](https://mill-build.com/mill/Library_Dependencies.html#_scalajs_dependencies).

Your example dependency should be written like this:

```scala
ivy"io.circe::circe-scalajs::0.14.6"
```

**Reply by roman-mibex-2** [reply]
Thanks. Yep, I didn't notice this subtle difference.
